src/test/java/baseball/ApplicationTest.java

```diff
@@ -28,6 +34,41 @@ class ApplicationTest extends NsTest {
         );
     }
 
+
+    //3개의 중복되지 않은 랜덤 숫자들을 잘 만드는 지
+    @Test
+    public void generateNumbers_ThreeUniqueNumbers() {
+        List<Integer> numbers = Application.generateNumbers();
+        assertEquals(3, numbers.size());
+        assertEquals(3, new HashSet<>(numbers).size());
+    }
+
+
+    //랜덤 숫자를 고정 후, 스트라이크와 볼, 낫싱을 잘 계산하는 지
+    @Test
```

Comment:
```suggestion
    @Test
    @DisplayName("랜덤 숫자를 고정 후, 스트라이크와 볼, 낫싱을 잘 계산하는 지")
```
@DisplayName 어노테이션을 통해서 해당 테스트가 무엇을 테스트하는 것인지 명시할 수 있습니다!

@DisplayName 어노테이션을 잘 작성하는 것이 중요한데요. 해당 테스트가 어떤 목표인 테스트인지 다른 사람이 봤을 때 잘 알 수 있도록 해야합니다.
또한, @DisplayName 어노테이션을 통해서 해당 함수가 어떤 기능을 하는 지 다른 사람들이 쉽게 파악할 수 있기 때문에 잘 적는 것이 중요합니다.

예시로 "랜덤 숫자를 고정 후, 스트라이크와 볼, 낫싱을 잘 계산하는 지" 같이 포괄적으로 적는 것이 아닌
"컴퓨터 번호가 1,2,3 이고, 유저 번호가 1,2,3일 때 3스트라이크를 반환한다." 와 같이 작성하면 좋을 것 같습니다!